external-crates/move/move-prover/interpreter-testsuite/tests/concrete_check/destroy.move

```diff
@@ -4,6 +4,7 @@ module 0x2::A {
         f2: u64,
     }
 
+    #[allow(unused_function)]
```

Comment:
Do you still need these with the move-prover/model suppression?

Reply:
This is yet another custom test runner in `UnitTestingConfig` and adding warning suppression seemed to make more sense than threading warning filters through another set of function calls. It's not like this was a call or two either, but I can investigate this if need be.